Framework/PythonInterface/test/python/plugins/algorithms/AbinsAdvancedParametersTest.py

```diff
@@ -0,0 +1,301 @@
+from __future__ import (division, print_function)
+import unittest
+import os
+from mantid.simpleapi import mtd, logger
+import numpy as np
+from mantid.simpleapi import Abins, DeleteWorkspace
+
+from AbinsModules import AbinsParameters, AbinsTestHelpers
+
+try:
+    from pathos.multiprocessing import ProcessingPool
+    PATHOS_FOUND = True
+except ImportError:
+    PATHOS_FOUND = False
+
+def old_modules():
+    """" Check if there are proper versions of  Python and numpy."""
+    is_python_old = AbinsTestHelpers.old_python()
+    if is_python_old:
+        logger.warning("Skipping AbinsBasicTest because Python is too old.")
+
+    is_numpy_old = AbinsTestHelpers.is_numpy_valid(np.__version__)
+    if is_numpy_old:
+        logger.warning("Skipping AbinsBasicTest because numpy is too old.")
+
+    return is_python_old or is_numpy_old
+
+
+def skip_if(skipping_criteria):
+    """
+    Skip all tests if the supplied function returns true.
+    Python unittest.skipIf is not available in 2.6 (RHEL6) so we'll roll our own.
+    """
+    def decorate(cls):
+        if skipping_criteria():
+            for attr in cls.__dict__.keys():
+                if callable(getattr(cls, attr)) and 'test' in attr:
+                    delattr(cls, attr)
+        return cls
+    return decorate
+
+
+@skip_if(old_modules)
+class AbinsAdvancedParametersTest(unittest.TestCase):
+
+    def setUp(self):
+
+        # set up input for Abins
+        self._Si2 = "Si2-sc_AbinsAdvancedParameters"
```

Comment:
I think this needs to be updated to `Si2-sc_ABINSAdvancedParameters` as the filename has not changed